InnerEye/ML/deep_learning_config.py

```diff
@@ -230,6 +230,13 @@ class WorkflowParams(param.Parameterized):
     monitoring_interval_seconds: int = param.Integer(0, doc="Seconds delay between logging GPU/CPU resource "
                                                             "statistics. If 0 or less, do not log any resource "
                                                             "statistics.")
+    regression_test_folder: Optional[Path] = \
+        param.ClassSelector(class_=Path, default=None, allow_None=True,
+                            doc="A path to a folder that contains a set of files. At the end of training and "
```

Comment:
Should we start the doc by "For testing purposes only"?

Reply:
not sure it's worth it, the field is already called "regression test"...